ceph.yaml

```diff
@@ -0,0 +1,349 @@
+package:
+  name: ceph
+  version: 19.2.1
```

Comment:
Needs an epoch field

Reply:
Don't know how I missed that all this time :-). Fixed now